components/calendar/src/cal/indian.rs

```diff
@@ -231,14 +222,6 @@ impl Indian {
     pub fn new() -> Self {
         Self
     }
-
-    fn days_in_year_direct(year: i32) -> u16 {
-        if Indian::is_leap_year(year, ()) {
-            366
-        } else {
-            365
-        }
-    }
```

Comment:
Question/Issue (here and elsewhere): Why is this function not reachable by the regular `days_in_year` getter on Date?

Reply:
These are only used to calculate days of the previous year.

Reply:
It's not a bug, calendar code is just very messy. There were two functions that did this, the other one is here: https://github.com/unicode-org/icu4x/blob/4076f827db5f725175392f91fe26ddb9dfb2ed8e/components/calendar/src/cal/indian.rs#L77